web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,139 @@
 ---
-sidebar_position: 2
+# Creating a New Project
 ---
 
-# Creating a New Project
+## Choosing an approach
+
+### Use build tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
+
+### Use dashboard upstream:
+
+- The simplest possible setup
+
+- Pantheon repo is your source of truth
+
+- GitHub is your default VCS and CircleCI is your CI provider then this approach will automatically default to both these options. For other options use the build tools approach
+
+### Installing using dashboard upstream
+- Create an empty upstream site:
+
+#### Via the Pantheon Dashboard at this link: 
+- https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65
+
+#### Or alternatively via terminus:
+
+```
+terminus site:create my-new-site "My New Site" empty --org='Decoupled Demo'
+```
+
+#### In the Pantheon Admin Dashboard:
+
+** Switch development mode to git
+
+** Click the upgrade button, and change the site plan to performance medium. 
```

Comment:
Check the styleguide for navigational elements within a UI: https://pantheon.io/docs/style-guide#bold